packages/dd-trace/src/plugins/util/test.js

```diff
@@ -16,14 +16,15 @@ const {
   GIT_COMMIT_AUTHOR_EMAIL,
   GIT_COMMIT_AUTHOR_NAME,
   GIT_COMMIT_MESSAGE,
-  CI_WORKSPACE_PATH
+  CI_WORKSPACE_PATH, CI_PIPELINE_URL
 } = require('./tags')
 const id = require('../../id')
 
 const { SPAN_TYPE, RESOURCE_NAME, SAMPLING_PRIORITY } = require('../../../../../ext/tags')
 const { SAMPLING_RULE_DECISION } = require('../../constants')
 const { AUTO_KEEP } = require('../../../../../ext/priority')
 const { version: ddTraceVersion } = require('../../../../../package.json')
+const { URL } = require('url')
```

Comment:
nitpick: let's move this import at the top, together with `path` and `fs` imports. We don't always enforce this, but it's a good pattern